services-js/commissions-search/package.json

```diff
@@ -12,6 +12,7 @@
     "react": "16.6.0",
     "react-apollo": "^2.1.9",
     "react-dom": "16.6.0",
+    "react-ga": "^2.5.6",
```

Comment:
Remove this, since it's not being used, and be sure to re-run `yarn install` so it gets taken out of `yarn.lock`